packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_token_burn_token.test.ts

```diff
@@ -0,0 +1,243 @@
+    const script = await providers.elliptic.script()
+    const promise = builder.tokens.burn({
+      amounts: [{ token: Number(idBTC), amount: new BigNumber(-2) }],
+      burnType: 0,
+      from: script,
+      variantContext: {
+        variant: 0,
+        context: {
+          stack: []
+        }
+      }
+    }, script)
+
+    await expect(promise).rejects.toThrow('The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received -200000000')
+  })
+
+  it('should throw an error if not enough tokens are available to burn', async () => {
+    const script = await providers.elliptic.script()
+    const txn = await builder.tokens.burn({
+      amounts: [{ token: Number(idBTC), amount: new BigNumber(15) }],
+      burnType: 0,
+      from: script,
+      variantContext: {
+        variant: 0,
+        context: {
+          stack: []
+        }
+      }
+    }, script)
+
+    // Ensure the created txn is correct
+    const promise = sendTransaction(testing.container, txn)
+
+    await expect(promise).rejects.toThrow(DeFiDRpcError)
+    await expect(promise).rejects.toThrow("DeFiDRpcError: 'BurnTokenTx: amount 0.00000000 is less than 15.00000000 (code 16)', code: -26")
+  })
+
+  it('should burnToken without context', async () => {
+    // Fund 100 BTC TOKEN
+    await testing.token.send({
+      address: await providers.getAddress(),
+      amount: 100,
+      symbol: 'BTC'
+    })
+    await testing.generate(1)
+
+    const script = await providers.elliptic.script()
+    const tokenBurn: TokenBurn = {
+      amounts: [{ token: Number(idBTC), amount: new BigNumber(15) }],
+      burnType: 0,
+      from: script,
+      variantContext: {
+        variant: 0,
+        context: script
+      }
+    }
+
+    const txn = await builder.tokens.burn(tokenBurn, script)
+
+    // Ensure the created txn is correct
+    const outs = await sendTransaction(testing.container, txn)
+    const encoded: string = OP_CODES.OP_DEFI_TX_TOKEN_BURN(tokenBurn).asBuffer().toString('hex')
+    const pubKey = await providers.ellipticPair.publicKey()
+    const address = Bech32.fromPubKey(pubKey, 'bcrt')
+    expect(outs).toStrictEqual([{
+      n: 0,
+      scriptPubKey: {
+        asm: expect.stringMatching(/^OP_RETURN 4466547846/),
+        hex: `6a${encoded}`,
+        type: 'nulldata'
+      },
+      tokenId: 0,
+      value: 0
+    }, {
+      n: 1,
+      scriptPubKey: {
+        addresses: [address],
+        asm: expect.any(String),
+        hex: expect.any(String),
+        reqSigs: 1,
+        type: 'witness_v0_keyhash'
+      },
+      tokenId: 0,
+      value: 99.9999904
+    }])
+
+    await testing.generate(1)
+
+    const attributes = await testing.rpc.masternode.getGov(attributeKey)
+    const burntKeyRegex = /^v0\/live\/economy\/consortium_members\/\d+\/\d+\/burnt$/
+    const keys: string[] = Object.keys(attributes.ATTRIBUTES)
+
+    // Verify the token is not burned to any consortium
```

Comment:
```suggestion
    // Verify that the burn action is not tied to any consortium member
```

I think this is more accurate?